.github/workflows/test.yml

```diff
@@ -5,12 +5,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ 12, 14, 16, 18 ]
+        node: [ 14, 16, 18, 20 ]
```

Comment:
12 should be kept, and since there's no engines field, older nodes should be added too.